### DISCUSSION TITLE
How to display blog page with include extension?

### DISCUSSION THREAD

**Discussion by tschurlo** [datenstrom/community#633] [Ask a question] [answered]
Hi all!

I really like the idea behind the Include extension, so I tried to use it to display a certain blog post with Include 0.8.5.
Blog post file names are per default created like `YYYY-MM-DD-blog-page`.
Unfortunately ,`[include YYYY-MM-DD-blog-page teaser]` does not work, as only "page not found" is displayed.
When I edit the file name to not include any dashes, numbers or underscores, e.g. `[include blogpage teaser]` the Include extension works as expected.

Please have a look, thx.
Kind regards,
M-Th.

**Comment by schulle4u** [comment, marked as answer]
Location arguments sllightly differ from directory paths. As for the blog extension, the date part will be removed from the resulting location. Example: The file `content/2-blog/2020-12-06-made-for-people.md` becomes `/blog/made-for-people`. And thats how the include extension works, just pass the clean location argument, e. g. `[include /blog/made-for-people teaser]`. You can try to open a blog page and copy the URL part after your website's domain name to get the right location argument. 

See also: [Adjusting content](https://datenstrom.se/yellow/help/adjusting-content)

**Reply by tschurlo** [reply]
Thank you for your quick reply!

Please consider updating the documentation of the include extension with this additional information.

**Reply by schulle4u** [reply]
Thanks for your feedback. I've updated the documentation with a more detailed description of the location argument, and also with another example for including blog pages. Hope this is less confusing now. 😀

**Reply by tschurlo** [reply]
Perfect, thx a lot!
